packages/medusa/src/migrations/1669032280563-payment-collection.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
question: Has this always been part of the PR? Seems misplaced

Reply:
Just saw your comment below. But should not be included now right?

Reply:
I need to re test, but i had an issue but in the end it was something else and I did not revert that one

Reply:
todo: should be removed